cmd/oras/internal/option/remote.go

```diff
@@ -77,14 +78,15 @@ func applyPrefix(prefix, description string) (flagPrefix, notePrefix string) {
 // Commonly used for non-unary remote targets.
 func (opts *Remote) ApplyFlagsWithPrefix(fs *pflag.FlagSet, prefix, description string) {
 	var (
-		shortUser     string
-		shortPassword string
-		shortHeader   string
-		flagPrefix    string
-		notePrefix    string
```

Comment:
Why are these called `short*` ? Unrelated to this change

Reply:
@sbdtu5498 is right. If `prefix` is not empty, short notation will not be enabled.